Expensify/App
# Use `GithubUtils` to access `octokit`

cc'ing @roryabraham for context/familiarity

### Details
Attempt to solve https://github.com/Expensify/App/runs/7866938557?check_suite_focus=true:

```
Fetching tag list from github...
/home/runner/work/_actions/Expensify/App/main/.github/actions/javascript/getDeployPullRequestList/index.js:45
    return octokit.repos.listTags({
                         ^

TypeError: Cannot read property 'listTags' of undefined
    at getTagsOrReleases (/home/runner/work/_actions/Expensify/App/main/.github/actions/javascript/getDeployPullRequestList/index.js:45:26)
    at Object.5847 (/home/runner/work/_actions/Expensify/App/main/.github/actions/javascript/getDeployPullRequestList/index.js:52:1)
    at __nccwpck_require__ (/home/runner/work/_actions/Expensify/App/main/.github/actions/javascript/getDeployPullRequestList/index.js:10798:43)
    at /home/runner/work/_actions/Expensify/App/main/.github/actions/javascript/getDeployPullRequestList/index.js:10815:18
    at Object.<anonymous> (/home/runner/work/_actions/Expensify/App/main/.github/actions/javascript/getDeployPullRequestList/index.js:10816:[12](https://github.com/Expensify/App/runs/7866938557?check_suite_focus=true#step:4:13))
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:[14](https://github.com/Expensify/App/runs/7866938557?check_suite_focus=true#step:4:15))
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
```

### Tests
Is this testable locally in advance?

### QA
1. Go to https://github.com/Expensify/App/actions/workflows/platformDeploy.yml
2. Open the first (most recent) "Merge pull request XXX from YYY"
3. Make sure the step "Post a GitHub comment when platforms are done building and deploying" completes successfully
<img width="970" alt="image" src="https://user-images.githubusercontent.com/2229301/185007827-18d1930c-3ffe-4b4e-9ea0-086de54fe0f7.png">
